### ISSUE TITLE
[Bug]: Token VBL carries from one token to another.

### ISSUE THREAD

**Issue by timgstewart** [RPTools/maptool#3234]
### Describe the Bug

If I generate VBL on a token, then drag a new token onto the map, the new token has the VBL from the previous token.  

### To Reproduce

1. Edit a token.
2. Generate some VBL
3. Exit the token editor (Ok button)
4. Drag a new token onto the map
5. Edit the new token.
6. Observe that it has the VBL already set to the VBL of the prior token.

### Expected Behaviour

I expected a new token would not have any VBL.

### MapTool Info

1.10.4, 1.11.0

### Desktop

Linux Ubuntu

### Additional Context

Issue exists on 1.10.4 and 1.11.0 so it probably has been there a while.

**Comment by kwvanderlinde** [comment]
There isn't actually any VBL on a new tokens _per se_. E.g., here I added VBL to the Elf, then dragged the Eagle onto the map and we can see the Eagle has no VBL:
![vbl-on-new-token](https://user-images.githubusercontent.com/7492219/144292750-a9fde88d-d92d-4054-ba05-d1c87257fc8f.png)

The problem is that the Edit Token dialog itself has some state that isn't being reset. I've noticed that the **VBL Preview** and the **Inverse VBL Color Selection** states are carried over from previous uses of the dialog. When the user clicks "OK", even if they never looked at the VBL tab, the VBL from the preview pane is then mistakenly applied to the token.

One more thing: even though the UI state for **Inverse VBL Color Selection** is carried over, it is desynced from the backend state. So this make it a bit annoying to generate inverted VBL for multiple tokens in a row.
